lib/utils/optOut.js

```diff
@@ -0,0 +1,7 @@
+export function setOptOut(optOut = true) {
+    if (optOut) {
+        localStorage.setItem('optout', optOut);
+    } else {
+        localStorage.removeItem('optout');
+    }
```

Comment:
let's avoid "else"s if it's possible
https://www.freecodecamp.org/forum/t/the-return-early-pattern/19364
```
 if (optOut) {
        localStorage.setItem('optout', optOut);
        return;
    }

 localStorage.removeItem('optout');
```